ci/verify_tensorrt.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import torch
+from bundle_custom_data import include_verify_tensorrt_list
+from monai.bundle import trt_export
+from verify_bundle import _find_bundle_file
+
+
+def verify_tensorrt(bundle_path: str, net_id: str, config_file: str, precision: str):
+    """
+    This function is used to verify if the checkpoint is able to torchscript, and
+    if "models/model.ts" is provided, it will be checked if is able to be loaded
+    successfully.
+
+    """
+    trt_model_path = os.path.join(bundle_path, f"models/model_trt_{precision}.ts")
+    trt_export(
+        net_id=net_id,
+        filepath=trt_model_path,
+        ckpt_file=os.path.join(bundle_path, "models/model.pt"),
+        meta_file=os.path.join(bundle_path, "configs/metadata.json"),
+        config_file=os.path.join(bundle_path, config_file),
+        precision=precision,
+        bundle_root=bundle_path,
+        dynamic_batchsize=(1, 1, 1),
+    )
+
+    _ = torch.jit.load(trt_model_path)
+
```

Comment:
Hi @binliunls , could you please help to check if the trt test here is enough?
As for the `dynamic_batchsize`, I hard coded to use (1, 1, 1). 
The function will check:
1. if bundle weights can be exported into trt model.
2. if the exported trt model can be loaded.

Reply:
Hi @yiheng-wang-nv ,
      The `dynamic_batchsize` support the None input with a warning.  Maybe you can just remove the `dynamic_batchsize` line.

Hi @Nic-Ma ,
       Do you think if it is necessary that we add a speadup ratio > 1 check for model computation?

Thanks,
Bin

Reply:
Hi @binliunls ,

Let's focus on the functionality tests for the first step, we can add more speed tests later.

Thanks.